src/main/webapp/scripts/suggest-script.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Gets executed when button is clicked
+ */
+window.onload = function() {
+  document.getElementById('make-suggestion').onclick = function() {
+    makeSuggestion();
+  };
+};
+
+/**
+ * Posts to server the suggestion made by the user
+ *
+ * Ignoring linter otherwise makeSuggestion is unsued but it is
+ * called in tutorials.html
+ */
+function makeSuggestion() {
+  const platElem = document.getElementById('platform').value;
```

Comment:
nitpick: `getElementById` is pretty good performance-wise, but a slight optimization would be to cache the value since you also reference it below. It's also closer to the variable name, but I would rename slightly to `const platformElement` - trying to shorten variable names usually just ends up making them harder to understand later 😄 

```
const platformElement = document.getElementById('platform');
$.post('/new-suggestion?platform=' + platformElement.value ...
platformElement.value = '';
```

Reply:
(same goes for `additionElement`)

Reply:
Cached the value and made variable name consistent.